### PR TITLE
Update gsettings-desktop-schemas to python 3.10

### PR DESCRIPTION
**Pull request by lukaso**
#### Description

Updates gsettings-desktop-schemas to python 3.10

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.6 21G115 arm64
Xcode 14.0.1 14A400

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: gnome/gsettings-desktop-schemas/Portfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ PortGroup           meson 1.0
 
 name                gsettings-desktop-schemas
 version             3.38.0
+revision            1
```
Comment (reneeotten) [inline gnome/gsettings-desktop-schemas/Portfile:8]: no need to increase the `revision` field as this is only a build dependency (see the relevant part of the [guide](https://guide.macports.org/#reference.keywords))

**Review by reneeotten — CHANGES_REQUESTED** [review]

**Review by reneeotten — CHANGES_REQUESTED** [review]
also, when correcting that please make sure you follow our guidelines for commit messages.

**Comment by lukaso** [comment]
> also, when correcting that please make sure you follow our guidelines for commit messages.

I've done it, but not really sure how the original message wasn't matching the commit guidelines (did this need a body?). Hope this is better. Thanks.

**Comment by mascguy** [comment]
> > also, when correcting that please make sure you follow our guidelines for commit messages.
> 
> I've done it, but not really sure how the original message wasn't matching the commit guidelines (did this need a body?). Hope this is better. Thanks.

No need for a body per se, but it's always fine to include additional details/clarification therein.

Looks good now though. Thanks Lukas!

**Review by mascguy — APPROVED** [review]

**Comment by reneeotten** [comment]
> I've done it, but not really sure how the original message wasn't matching the commit guidelines (did this need a body?). 

thanks @lukaso ! The commit guidelines are: `<portname>: short description`, which didn't match your original commiet message which was the same as the title of this PR. A body isn't necessary here as it doesn't provide information that cannot be easily deduced from the actual code change.

**Comment by lukaso** [comment]
> The commit guidelines are: `<portname>: short description`

When I looked at the guidelines, this is listed as a suggestion: "List any modified ports. **A good way to do this is** to put their names first, followed by a colon". Bolding added by me. https://trac.macports.org/wiki/CommitMessages

If that is the way it should be done, I would recommend making that explicit. That's why I felt my message did meet the guidelines which were "List any modified ports", even if it didn't meet the suggestion.

BTW: I only mention this to improve things. I'm fine with the guideline being stricter :)

**Pull request merged**